apps/management/src/client/components/IdentitiesList.tsx

```diff
@@ -1,10 +1,10 @@
-import type { Manager } from '@3id/manager'
+import type { Manager } from '@3id/did-manager'
 import { Avatar, Box, Text } from 'grommet'
 import { useMemo } from 'react'
 
 import Button from '../../components/Button'
 import { formatDID, getImageSrc, longFormatDID } from '../../data/idx'
-import avatarPlaceholder from '../../images/avatar-placeholder.png'
+// import avatarPlaceholder from '../../images/avatar-placeholder.png'
```

Comment:
Next supports importing media like this but returns an object with metadata, so to get the URL you probably need to use `avatarPlaceholder.src`, see https://github.com/ceramicstudio/self.id/blob/main/apps/self.id/src/auth.ts#L10 for example

Reply:
ah ok 👍